bundles/org.openhab.binding.sonos/src/main/resources/ESH-INF/thing/SYMFONISK.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="sonos"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Player SYMFONISK Thing Type -->
+	<thing-type id="SYMFONISK">
+		<label>SYMFONISK</label>
+		<description>Represents IKEA SYMFONISK speaker</description>
+
+		<channels>
+			<channel id="add" typeId="add" />
+			<channel id="alarm" typeId="alarm" />
+			<channel id="alarmproperties" typeId="alarmproperties" />
+			<channel id="alarmrunning" typeId="alarmrunning" />
+			<channel id="control" typeId="system.media-control" />
+			<channel id="currentalbum" typeId="currentalbum" />
+			<channel id="currentalbumart" typeId="currentalbumart" />
+			<channel id="currentalbumarturl" typeId="currentalbumarturl" />
+			<channel id="currentartist" typeId="system.media-artist" />
+			<channel id="currenttitle" typeId="system.media-title" />
+			<channel id="currenttrack" typeId="currenttrack" />
+			<channel id="shuffle" typeId="shuffle" />
+			<channel id="repeat" typeId="repeat" />
+			<channel id="favorite" typeId="favorite" />
+			<channel id="led" typeId="led" />
+			<channel id="localcoordinator" typeId="localcoordinator" />
+			<channel id="mute" typeId="system.mute" />
+			<channel id="notificationsound" typeId="notificationsound" />
+			<channel id="playlist" typeId="playlist" />
+			<channel id="clearqueue" typeId="clearqueue" />
+			<channel id="playlinein" typeId="playlinein" />
+			<channel id="playqueue" typeId="playqueue" />
+			<channel id="playtrack" typeId="playtrack" />
+			<channel id="playuri" typeId="playuri" />
+			<channel id="publicaddress" typeId="publicaddress" />
+			<channel id="radio" typeId="radio" />
+			<channel id="remove" typeId="remove" />
+			<channel id="restore" typeId="restore" />
+			<channel id="restoreall" typeId="restoreall" />
+			<channel id="save" typeId="save" />
+			<channel id="saveall" typeId="saveall" />
+			<channel id="snooze" typeId="snooze" />
+			<channel id="standalone" typeId="standalone" />
+			<channel id="state" typeId="state" />
+			<channel id="stop" typeId="stop" />
+			<channel id="tuneinstationid" typeId="tuneinstationid" />
+			<channel id="volume" typeId="system.volume" />
+			<channel id="zonegroupid" typeId="zonegroupid" />
+			<channel id="zonename" typeId="zonename" />
+			<channel id="coordinator" typeId="coordinator" />
+			<channel id="sleeptimer" typeId="sleeptimer" />
+			<channel id="currenttransporturi" typeId="currenttransporturi" />
+			<channel id="currenttrackuri" typeId="currenttrackuri" />
+		</channels>
+
+		<properties>
+			<property name="vendor">SONOS</property>
```

Comment:
Is the vendor Sonos or IKEA...?

Reply:
It is sold by IKEA. So IKEA could make sense at this place.
But you get Sonos everywhere in the feature XML, not IKEA.

Reply:
Well, in the description you also call it "IKEA SYMFONISK", which is imho correct. So IKEA is in the product name and they are selling it - it is only Sonos who delivery the technology for it. So I'd suggest to change the vendor here.